build.gradle

```diff
@@ -228,11 +240,17 @@ task licenseFormatAll
 subprojects { p -> licenseFormatAll.dependsOn("${p.path}:licenseFormat") }
 subprojects.each { remapJar.dependsOn("${it.path}:remapJar") }
 
+sourceSets {
+	testmod
+}
+
 dependencies {
 	afterEvaluate {
 		subprojects.each {
 			compile project(path: ":${it.name}", configuration: "dev")
 			include project("${it.name}:")
+
+			testmodCompile project("${it.name}:").sourceSets.testmod.output
```

Comment:
```suggestion
			testmodImplementation project("${it.name}:").sourceSets.testmod.output
```

Reply:
not a big issues, as comple is used in other places.